mobile-app/app/screens/AppNavigator/screens/Loans/components/CollateralizationRatioDisplayV2.tsx

```diff
@@ -97,18 +111,8 @@ export function CollateralizationRatioDisplayV2(
       {props.showProgressBar && (
         <View style={tailwind("mt-2.5")}>
           <Progress.Bar
-            progress={
-              normalizedNextFactor.lt(0) &&
-              props.collateralizationRatio.length > 0
-                ? 1
-                : normalizedNextFactor.toNumber()
-            }
-            color={getColor(
-              normalizedNextFactor.lt(0) &&
-                props.collateralizationRatio.length > 0
-                ? "green-v2"
-                : ratioTextColor
-            )}
+            progress={isReadyForLoan ? 1 : normalizedNextFactor.toNumber()}
```

Comment:
@fullstackninja864 has a use case where normalizedNextFactor is negative value

Reply:
another issue for my borrow loan use case is that, my vault has collateral, but the progress bar will always have full width based on your current condition, regardless of my col ratio

https://user-images.githubusercontent.com/40191153/195475952-4544ff82-52ae-46ae-b1e7-7c5dd7201387.mp4

Reply:
Thanks for the info. I didn't know these things, I reverted any changes related to progress bar. I kept my change minimal in this PR and shouldn't affect other screens unless you pass the new props I added: `isReadyForLoan, customReadyText`

Reply:
no worries, which is why i am sharing the info now ;)